corehq/apps/users/models.py

```diff
@@ -776,6 +776,8 @@ def wrap(cls, data, should_save=False):
         couch_user = super(CouchUser, cls).wrap(data)
         if should_save:
             couch_user.save()
+
+        couch_user._original_domains = set(couch_user.get_domains())
```

Comment:
There is not `__init__`, `wrap` is what get's called to create the python model from the json. 

Private fields shouldn't get saved to the DB